Explorer/Assets/Scripts/Utility/Primitives/DefaultMaterial.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+namespace Utility.Primitives
+{
+    public static class DefaultMaterial
+    {
+        private static Material cached;
+
+        private static readonly Shader CACHED_SHADER = Shader.Find("Universal Render Pipeline/Lit");
```

Comment:
This will not work in the build. We either need to add the shader variant; or change it to `DCL/Universal Render Pipeline/Lit` (since that shader is included by the material added in Resources)